nsv13/code/modules/overmap/weapons/projectiles_fx.dm

```diff
@@ -563,12 +563,31 @@ Misc projectile types, effects, think of this as the special FX file.
 	tracer_type = /obj/effect/projectile/tracer/disabler
 	muzzle_type = /obj/effect/projectile/muzzle/disabler
 	impact_type = /obj/effect/projectile/impact/disabler
+	relay_projectile_type = /obj/item/projectile/beam/laser/heavylaser/phaser/relayed
+
+/obj/item/projectile/beam/laser/heavylaser/phaser/relayed
+	projectile_piercing = PASSGLASS|PASSGRILLE|PASSTABLE
+	flag = "laser"
+	damage = 40 //let's give them a chance to live, instead of smiting them with the full wraith of the Enterprise 
```

Comment:
```suggestion
	damage = 80 //let's give them a chance to live, instead of smiting them with the full wraith of the Enterprise 
```
A bit more is fine, at least make sure you change the red laser to do less damage than this one if you keep it lower.